local/snapshot.go

```diff
@@ -116,11 +117,7 @@ func (ln *localNetwork) SaveSnapshot(ctx context.Context, snapshotName string) (
 		nodeConfig := node.config
 		// depending on how the user generated the config, different nodes config flags
 		// may point to the same map, so we made a copy to avoid always modifying the same value
-		nodeConfigFlags := make(map[string]interface{})
-		for fk, fv := range nodeConfig.Flags {
-			nodeConfigFlags[fk] = fv
-		}
-		nodeConfig.Flags = nodeConfigFlags
+		nodeConfig.Flags = maps.Clone(nodeConfig.Flags)
```

Comment:
So, this change is functionally equivalent to what was here before, but it's actually a no-op. Can we remove or fix this?

Reply:
It is not a no-op. It is creating a new map. To solve situation where different nodes share the same map. Probably will require a fix but I think not here.

Reply:
`To solve situation where different nodes share the same map.` 

Ok if that's the case then it's not a no-op. This change is semantically the same as before so should be fine.